src/themes/store/components/Heading.js

```diff
@@ -16,7 +16,8 @@ const Heading = {
       textStyle: 'h1',
     },
     lightCaption: {
-      textStyle: 'h1',
+      textStyle: 'lightCaption',
+      fontWeight: 400,
```

Comment:
for some reason, I had to apply it here because applying it inside `textStyles.js` wouldn't change anything, even if I removed `...headingsBase` under `lightCaption`

Reply:
yeah, i had issues with `textStyle` as well. I think there's actually some open bugs in Chakra-UI for `textStyle` not working in some cases :'(